doc/static_discovery_configuration_guide.md

```diff
@@ -0,0 +1,808 @@
+sudo chmod 751 /etc/harmony-ap/tls-truststore.jks
+```
+
+#### 3.6.2 Access Point 2
+
+The sign certificate of the Access Point 1 (`org1_gw`) is imported using the admin UI by following the steps below:
+
+1. Click PMode and then Parties.
+2. Select the party `org1_gw` and click Edit.
+3. Click the Import button in the Certificate section.
+4. Select the certificate to be imported.
+5. Click OK and then click Save.
+
+The TLS certificate is imported on the command line. The TLS truststore doesn't exist yet and it will be created when 
+the TLS certificate of the Access Point 1 (`org1_gw`) is imported. Therefore, generate a secure password for the TLS truststore 
+(`tls_truststore_password`):
+
+```
+openssl rand -base64 12
+```
+
+**Note:** Write down the password (`tls_truststore_password`). You will need it in step 3.7.
+
+Then, import the TLS certificate of Access Point 1 (`org1_gw`):
+
+```
+sudo keytool -import -alias org1_gw -file org1_tls_certificate.cer -keystore /etc/harmony-ap/tls-truststore.jks -storepass <tls_truststore_password>
+```
+
+Set the file permissions of the new TLS truststore file:
+
+```
+sudo chown harmony-ap:harmony-ap /etc/harmony-ap/tls-truststore.jks
+sudo chmod 751 /etc/harmony-ap/tls-truststore.jks
+```
+
+### 3.7 Configure One-Way SSL
+
+This step requires shell access to the host.
+    
+On Access Points 1 (`org1_gw`) and Access Point 2 (`org2_gw`), create the `/etc/harmony-ap/clientauthentication.xml` 
+file with the following content:
+
+```
+<http-conf:tlsClientParameters disableCNCheck="true" secureSocketProtocol="TLSv1.2"
+                               xmlns:http-conf="http://cxf.apache.org/transports/http/configuration"
+                               xmlns:security="http://cxf.apache.org/configuration/security">
+    <security:trustManagers>
+        <security:keyStore type="JKS" password="<tls_truststore_password>"
+                           file="/etc/harmony-ap/tls-truststore.jks"/>
+    </security:trustManagers>
+</http-conf:tlsClientParameters>
+```
+
+**Note:** Use the password (`tls_truststore_password`) created in step 3.6.
+
+Set the file permissions of the new `/etc/harmony-ap/clientauthentication.xml` file:
+
+```
+sudo chown harmony-ap:harmony-ap /etc/harmony-ap/clientauthentication.xml
+sudo chmod 751 /etc/harmony-ap/clientauthentication.xml
+```
+
+### 3.8 Apply Changes
+
+This step requires shell access to the host.
+
+On Access Points 1 (`org1_gw`) and Access Point 2 (`org2_gw`), restart the `harmony-ap` service to apply the 
+configuration changes:
+
+```
+sudo systemctl restart harmony-ap
+```
+
+### 3.9 Send Test Message
+
+Send a request to Access Point 1 (`org1_gw`) using the curl command below. The request (`submitRequest.xml`) can be downloaded [here](configuration_examples/static_discovery/submitRequest.xml). The content inside the payload's `value` element must be [base64 encoded](https://www.base64encode.org/).
```

Comment:
It might be better to link to the raw file in case someone just wants to grab the link and run wget or curl with it. Currently, it would give the github HTML page.

Reply:
I updated the link to point to the raw file.